.github/workflows/e2e_tests.yml

```diff
@@ -5,21 +5,34 @@ on: [push, workflow_dispatch]
 jobs:
   e2e_build_and_test:
     name: Build Container and Run E2E Tests
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     env:
       WP_VERSION: 6.1.0
       TRAVIS_MARIADB_VERSION: 10.8.3
     steps:
-      - uses: actions/checkout@v2
+      - name: Set Git to use HTTPS instead of SSH
+        run: git config --global url.https://github.com/.insteadOf git://github.com/
+      - uses: actions/checkout@v3
         with:
           submodules: recursive
       - uses: actions/setup-node@v2
         with:
-          node-version: '10.16.0'
-      - run: git config --global url.https://github.com/.insteadOf git://github.com/
-      - run: npm ci
-      - run: npm run dist
-      - run: npm run test:e2e-docker-up
-      - run: npm run test:e2e-docker-ping
-      - run: npm run test:e2e
-      - run: npm run test:e2e-docker-down
+            node-version-file: '.nvmrc'
+      - name: Kill process running on port 8084 to unblock it
```

Comment:
Do you know what is running on 8084 in test? Can we use another port instead of killing the process?

Reply:
I think this is the `mono-xsp4` service? Taking suggestion from [here](https://github.com/actions/runner-images/issues/2821#issuecomment-789512742). What do you think about the following?

```
- name: disable and stop mono-xsp4.service
  run: |
       sudo systemctl stop mono-xsp4.service 
       sudo systemctl disable mono-xsp4.service
```

This way, we know what we are disabling and in the event Ubuntu changes the service port (ie. to 8080 in the future or something), we could still disable the service regardless of the port.

Reply:
Yep, that's it! Interesting! Let's try that. I guess the "kill the thing on this port" approach is more straightforward and future-proof but I'd rather have it break so we can know what's going on. Adding this now.

Reply:
I gave it a try but it didn't work out of the box. Let's go with the "kill 8084" approach for now. WDYT?

Reply:
Sounds good! Thanks for trying. Maybe that's why there were 2 thumbs down in the other thread 😆

Reply:
![](https://i.gifer.com/sdZ.gif)